api/pom.xml

```diff
@@ -18,6 +18,10 @@
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Why is this depends needed? 
You could just exclude commons logging like you do below 
and have the client add whatever they might need

Reply:
It's needed to give our transitive dependency the ability to log. Since they use `commons-logging`, it's likely that they require its interface. slf4j bridge does that: http://www.slf4j.org/legacy.html

Reply:
ok